### PR TITLE
[hi] Tweak grammar for localized task / tutorial prerequisites

### PR DESCRIPTION
**Pull request by Vikash-8090-Yadav**
Improved the grammar

### CODE REVIEW

File: content/hi/includes/task-tutorial-prereqs.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को 
-समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सकें। हमारी सलाह है की इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करे जो कि कंट्रोल प्लेन 
-होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप 
-[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
-की मदद से वह बना सकते है या आप नीचे दिए हुए इन दो कुबरनेट्स प्लेग्राउंड का इस्तेमाल कर सकते हैं:
-
-* [Katacoda](https://www.katacoda.com/courses/kubernetes/playground)
-* [प्ले विथ कुबेरनेट्स](https://labs.play-with-k8s.com/)
+आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को 
+समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सकें। हमारी सलाह है की इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करें जो कि कंट्रोल प्लेन
```
Comment (kundan2707) [inline content/hi/includes/task-tutorial-prereqs.md:2]: समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। हमारी सलाह है कि

**Comment by Vikash-8090-Yadav** [comment]
Done with the changes. Thanks @kundan2707

File: content/hi/includes/task-tutorial-prereqs.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
-* [Katacoda](https://www.katacoda.com/courses/kubernetes/playground)
-* [प्ले विथ कुबेरनेट्स](https://labs.play-with-k8s.com/)
+आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को 
+समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। हमारी सलाह है कि इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करें जो कि कंट्रोल प्लेन 
+होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप 
+[minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
+की मदद से वह बना सकते है या आप नीचे दिए हुए इन दो कुबरनेट्स प्लेग्राउंड का उपयोग कर सकते हैं:
+
+* [Katacoda](https://www.katacoda.com/courses/kubernetes/playground)
+* [प्ले विथ कुबेरनेट्स](http://labs.play-with-k8s.com/)
```
Comment (dipesh-rawat) [inline content/hi/includes/task-tutorial-prereqs.md:8]: We recently merged PR[#41665](https://github.com/kubernetes/website/pull/41665/files) for the main website (equivalent 'hi' PR[#41917](https://github.com/kubernetes/website/pull/41917/files) ) to change this link to use secure HTTPS. However, I'm wondering why the link in this particular PR has been altered to an insecure 'HTTP' again.

**Comment by kundan2707** [comment]
> Done with the changes. Thanks @kundan2707
Make sure to squash all commits to a single once after resolving all comments from all reviewers.

**Comment by sftim** [comment]
/label tide/merge-method-squash
/retitle Tweak grammar for localized (hi) task / tutorial prerequisites

**Comment by sftim** [comment]
/reopen
Sorry, misclick!

File: content/hi/includes/task-tutorial-prereqs.md [hunk 3]
```diff
@@ -1,8 +1,8 @@
 आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को 
-समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सकें। हमारी सलाह है की इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करे जो कि कंट्रोल प्लेन 
+समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। हमारी सलाह है कि इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करें जो कि कंट्रोल प्लेन 
 होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप
```
Comment (aj11anuj) [inline content/hi/includes/task-tutorial-prereqs.md:3]: This one is more appropriate

समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। हमारी सलाह है कि इस ट्यूटोरियल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करें जो कि कंट्रोल प्लेन
होस्ट की तरह कार्य ना करे । अगर आपके पास पहले से क्लस्टर नही है, तो आप

File: content/hi/includes/task-tutorial-prereqs.md [hunk 4]
```diff
@@ -1,8 +1,8 @@
 आपको कुबरनेट्स क्लस्टर की ज़रूरत पड़ेगी और क्यूब सीटीएल कमांड लाइन साधन को 
-समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सकें। हमारी सलाह है की इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करे जो कि कंट्रोल प्लेन 
+समनुरूप करना होगा ताकि वो आपके क्लस्टर के साथ संवाद कर सके। हमारी सलाह है कि इस टुटोरिअल को क्लस्टर में रन करने के लिए कम से कम दो नोड का इस्तेमाल करें जो कि कंट्रोल प्लेन 
 होस्ट के तरह ना एक्ट करे। अगर आपके पास पहले से क्लस्टर नही है, आप 
 [minikube](https://minikube.sigs.k8s.io/docs/tutorials/multi_node/)
-की मदद से वह बना सकते है या आप नीचे दिए हुए इन दो कुबरनेट्स प्लेग्राउंड का इस्तेमाल कर सकते हैं:
+की मदद से वह बना सकते है या आप नीचे दिए हुए इन दो कुबरनेट्स प्लेग्राउंड का उपयोग कर सकते हैं:
 
 * [Katacoda](https://www.katacoda.com/courses/kubernetes/playground)
 * [प्ले विथ कुबेरनेट्स](https://labs.play-with-k8s.com/)
```
Comment (aj11anuj) [inline content/hi/includes/task-tutorial-prereqs.md:8]: I think either "Play with Kubernetes" needs to be written in English or "Katacoda", "minikube" needs be written in hindi

**Review by aj11anuj — CHANGES_REQUESTED** [review]
Here are my thoughts, and some suggestions

**Comment by niranjandarshann** [comment]
@Vikash-8090-Yadav  Could you please squash your commits: https://kubernetes.io/docs/contribute/new-content/open-a-pr/#squashing-commits.

**Pull request closed without merging**